packages/open-vm-tools/open-vm-tools.spec

```diff
@@ -1,7 +1,7 @@
-%global buildver 18090558
+%global buildver 18557794
 
 Name: %{_cross_os}open-vm-tools
-Version: 11.3.0
+Version: 11.3.5
 Release: 1%{?dist}
```

Comment:
nit: you should be able to remove the `%exclude %{_cross_sbindir}/mount.vmhgfs` line in `%files` since that tool was removed upstream.